static/app/components/profiling/profilingTransactionHovercard.tsx

```diff
@@ -201,7 +169,7 @@ function ContextDetail(props: ContextDetailProps) {
 
   return (
     <Flex column gap={space(1)}>
-      {title && <ContextTitle>{title}</ContextTitle>}
+      {title && <UppercaseTitle>{title}</UppercaseTitle>}
```

Comment:
the previous component here `<ContextTitle/>` had some global styles coming in... opting to localize it here to ensure this component can consistently be used. --- we may end up using the body of the hovercard in the slowest transaction panel

i was experimenting with it here, hence the change to the title comp